maitake/src/wait/cell.rs

```diff
@@ -271,17 +367,14 @@ pub(crate) mod test_util {
                     None => return Poll::Ready(()),
                 };
 
-                let res = test_dbg!(this.task.poll_wait(cx.waker()));
+                let res = this.task.wait();
```

Comment:
I admit to not totally understanding this test, but I've more or less done a 1:1 port with the new, only-`wait()` API.